plugins/reportal/src/azkaban/reportal/util/ReportalUtil.java

```diff
@@ -57,4 +62,63 @@ public static IStreamProvider getStreamProvider(String fileSystem) {
 
     return sortedNodes;
   }
+
+  /**
+   * Get runtime variables to be set in unscheduled mode of execution.
+   * Returns empty list, if no runtime variable is found
+   *
+   * @param variables
+   * @return
+   */
+  public static List<Variable> getRunTimeVariables(
+    Collection<Variable> variables) {
+    List<Variable> runtimeVariables =
+      ReportalUtil.getVariablesByRegex(variables, "(?!(^"
```

Comment:
@evlstyle Idea in this regex is to match properties which does not match REPORTAL_CONFIG_PREFIX regex. We have used negative look around technique for this. Please refer to http://www.regular-expressions.info/lookaround.html

@davidzchen created constant.